docs/index.mdx

```diff
@@ -1,215 +1,63 @@
-            experiment_prefix="sample-experiment", # The name of the experiment
-            metadata={"version": "1.0.0", "revision_id": "beta"}, # Metadata about the experiment
-            max_concurrency=4,  # Add concurrency.
-        )
-
-        # Analyze the results via the UI or programmatically
-        # If you have 'pandas' installed you can view the results as a
-        # pandas DataFrame by uncommenting below:
-
-        # experiment_results.to_pandas()
-
-`,
-    typescript`
-import { Client } from "langsmith";
-import { EvaluationResult, evaluate } from "langsmith/evaluation";
-
-      const client = new Client();
-
-      // Define dataset: these are your test cases
-      const datasetName = "Sample Dataset";
-      const dataset = await client.createDataset(datasetName, {
-        description: "A sample dataset in LangSmith.",
-      });
-      await client.createExamples({
-        inputs: [
-          { postfix: "to LangSmith" },
-          { postfix: "to Evaluations in LangSmith" },
-        ],
-        outputs: [
-          { response: "Welcome to LangSmith" },
-          { response: "Welcome to Evaluations in LangSmith" },
-        ],
-        datasetId: dataset.id,
-      });
-
-      // Define your evaluator(s)
-      const exactMatch = async ({ outputs, referenceOutputs }: {
-        outputs?: Record<string, any>;
-        referenceOutputs?: Record<string, any>;
-      }): Promise<EvaulationResult> => {
-        return {
-          key: "exact_match",
-          score: outputs?.response === referenceOutputs?.response,
-        };
-      };
-
-      // Run the evaluation
-      const experimentResults = await evaluate(
-        (inputs: { postfix: string }) => ({ response: \`Welcome $\{inputs.postfix\}\` }),
-        {
-          data: datasetName,
-          evaluators: [exactMatch],
-          metadata: { version: "1.0.0", revision_id: "beta" },
-          maxConcurrency: 4,
-        }
-      );
-    `,
-
-]}
-groupId="client-language"
-/>
-
-- Click the link printed out by your evaluation run to access the LangSmith experiments UI,
-  and explore the results of your evaluation.
-- Learn more about evaluation in the [tutorials](./evaluation/tutorials), [conceptual guide](./evaluation/concepts), and [how-to guides](./evaluation/how_to_guides/index.md).
+## Observability
+
+Observability is important for any software application, but especially so for LLM applications. LLMs are non-deterministic by nature, meaning they can produce unexpected results. This makes them trickier than normal to debug.
+
+This is where LangSmith can help! LangSmith has LLM-native observability, allowing you to get meaningful insights from your application. LangSmith’s observability features have you covered throughout all stages of application development - from prototyping, to beta testing, to production. 
+
+- Get Started by [adding tracing](./observability/tutorials/observability) to your application.
```

Comment:
```suggestion
- Get started by [adding tracing](./observability/tutorials/observability) to your application.
```

Reply:
this should link to the new observability quickstart in this PR https://langsmith-docs-git-isaac-observabilitytutorials-langchain.vercel.app/observability